jsonnet/main.jsonnet

```diff
@@ -96,6 +96,35 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                },
                prometheusAdapter+:: {
                  prometheusURL: 'https://prometheus-k8s.openshift-monitoring.svc:9091',
+                 config: {
```

Comment:
In jsonnet you can actually override deeply nested fields with `+:`. In your case since you only want to modify the node queries, you can do something like that:

```
config+: {
  resourceRules+: {
    cpu+: {
      nodeQuery: 'your change',
    },
  },
},
```

Reply:
thanks for the tip @dgrisonnet , that definitely reduces the code duplication here, I have pushed the changes PTAL.